jmeter/plugins/jmeter-backendlistener-maestro/README.md

```diff
@@ -0,0 +1,63 @@
+Maestro Backend Listener for JMeter
+----
+Maestro Backend Listener is a JMeter plugins that sends sample results to Maestro API.
+
+
+## Sample Result Metrics
+
+- datetime
+- elapsed
+- label
+- datetime
+- elapsed
+- label
+- response_code
+- response_message
+- thread_name
+- data_type
+- success
+- failure_message
+- bytes
+- sent_bytes
+- grp_threads
+- all_threads
+- url
+- latency
+- idle_time
+- connect
+
+
+## Development/Build Environment
+### Prerequisites
+You need to have the following programs installed in your system:
+
+- JDK 1.8
+- Maven 3.8.x
+
+
+### Create JAR package
+
+To create the JMeter BackendListener Maestro plugin JAR package run:
+
+```bash
+mvn package
+```
+
+When the packaging process finishes, the .jar file will be available on the directory `./target`.
+
+
+## Plugin Setup
+The main use case for this plugin is to be used with Maestro. The maestro agent will be responsible for injecting the plugin and configuring it for each test it runs. 
+
+If you have a need to use it outside the maestro management scope please follow the steps below:
+![JMeter backendlistener maestro window](./docs/img/jmeterbackendlistenermaestrowindow.png)
+
+1. Create the JMeter BackendListener Maestro plugin JAR package and then put it into JMeter's lib/ext directory.
+2. Add JMeter BackendListener Plugin to your test plan.
+3. Select `com.farfetch.jmeter.backendlistener.maestro.MaestroBackendClient` Backend Listener Implementation.
+4. Set the parameters `maestroUrl`, `maestroAuthToken` and `maestroRunId` accordingly.
```

Comment:
maybe we could have those steps based on the already published package and then the other ones 'How to build a package'. What do you think?

Reply:
With the latest changes on the readme format, I added the first step to the _installation_ section. The following 3 steps belong now to the _usage_ section. Now we have segregation between the build/packaging steps and the configuration/usage of the plugin.